packages/client/src/workflow-client.ts

```diff
@@ -946,6 +926,31 @@ export class WorkflowClient {
       followRuns: options?.followRuns ?? true,
     });
   }
+
+  /**
+   * List workflows by given `query`.
+   *
+   * See more info on how to build the query in the Temporal docs: https://docs.temporal.io/visibility
+   *
+   * This API requires Elasticsearch based visibility as of server version 1.18.
+   */
+  public async *list(query: string, options?: ListOptions): AsyncIterable<WorkflowExecutionInfo> {
```

Comment:
Should query be a list option or at least optional?

Reply:
Good question... is it optional on the server?

Reply:
think so

<img width="847" alt="image" src="https://user-images.githubusercontent.com/251288/198739015-e7349208-1e10-4690-96f1-3cbae3df069c.png">

Reply:
Looks like it's optional on the server..

Reply:
Made `query` optional.